### ISSUE TITLE
Not working with @import inside sass file, and no aliases

### ISSUE THREAD

**Issue by VitaliyDU** [Feiyang1/awesome-typed-sass-modules#15]
File to import not found or unreadable: ~bourbon/core/bourbon.

/sources is alias
Error: File to import not found or unreadable: /sources/sass/global.

**Comment by Feiyang1** [comment]
Can you please provide some more detail? What command were you running?

**Comment by VitaliyDU** [comment]
atsm sources -p **/*.sass -o types

**Comment by VitaliyDU** [comment]
> atsm sources -p **/*.sass -o types
> In global.sass file i have `@import "~bourbon/core/bourbon"`, its without alias, import bourbon.scss from node_modules.

**Comment by hally9k** [comment]
Yeah, I'm getting an error when using this on scss files with @import statements with ~ prefix.
`File to import not found or unreadable`

**Comment by Feiyang1** [comment]
It appears ~ prefix is not part of the sass spec, but something sass-loader added to work with webpack.
This tool is only intended to support standard sass features, so I don't think ~ prefix is going to be supported.

**Comment by VitaliyDU** [comment]
https://github.com/webpack-contrib/sass-loader#imports, in that case your tool is not fully support sass, and have seriously restriction.

**Comment by Feiyang1** [comment]
I don't think it's fair to say the tool doesn't fully support sass since ~ is not part of the sass standard. It's a webpack specific thing and depends on webpack resolving engine AFAIK. 

This tool delegated all sass related functionalities (parsing, import resolution) to node-sass. and I think they fully support sass. 

That being said, I found an API in node-sass that allows you to plug in your own import resolution logic. And there is already a npm module that provides this logic (https://www.npmjs.com/package/node-sass-tilde-importer). You can take a look at it and see if it works for you. PR is welcomed. Otherwise I will take a look in a few day when I get time.

**Comment by Feiyang1** [comment]
Released 1.2.0 with ~ import support. Can you please try the new version?

**Comment by vitaliy-du** [comment]
Path with .extension **@ import "~bourbon/core/bourbon.scss"** is worked, without extension **@ import "~bourbon/core/bourbon"** **Error: File to import not found or unreadable: .../node_modules/bourbon/core/bourbon/index**. The @ import from root or relative not worked too.
